compiler/noirc_evaluator/src/errors.rs

```diff
@@ -93,7 +93,10 @@ impl From<SsaReport> for FileDiagnostic {
                 let message = bug.to_string();
                 let (secondary_message, call_stack) = match bug {
                     InternalBug::IndependentSubgraph { call_stack } => {
-                        ("There is no path from the output of this brillig call to either return values or inputs of the circuit, which creates an independent subgraph. This is quite likely a soundness vulnerability".to_string(),call_stack)
+                        ("There is no path from the output of this brillig call to either return values or inputs of the circuit, which creates an independent subgraph. This is quite likely a soundness vulnerability".to_string(), call_stack)
+                    }
+                    InternalBug::UncheckedBrilligCall { call_stack } => {
+                        ("There is no assert checking this brillig call's inputs against its return values in some way. This should be done to prevent potential soundness vulnerabilities".to_string(), call_stack)
```

Comment:
Would this be more accurate since constraints other than `assert`'s are checked? It would agree with the wording in other messages.
```suggestion
                        ("There is no constraint between this brillig call's inputs and its return values. This should be done to prevent potential soundness vulnerabilities".to_string(), call_stack)
```